test/tools/WebListener/Controllers/ResumeController.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
```

Comment:
Please add blank line after copyright header.
(I see the same in other files 😕 )

Reply:
fixed. And consider opening an issue.